.claude/references/proposed-patterns/pattern-skill-bootstrapping.md

```
@@ -0,0 +1,181 @@
+    |
+    v
+Agent produces: artifact + rules + self-check
+    |
+    v
+Self-check passes? --NO--> Agent fixes artifact, re-runs check
+    |
+   YES
+    |
+    v
+Rules are visible to future agents
+    |
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)